docs/source/examples/06-distributed-post/00-distributed_total_disp.py

```diff
@@ -3,10 +3,13 @@
 
 Distributed post without client connection to remote processes
```

Comment:
@jose-henriquez-roa could you please rename this example to: "Post processing of displacement on distributed processes"